managing_providers/_topics/automation_management_providers.md

```diff
@@ -6,10 +6,14 @@ In {{ site.data.product.title_short }}, an automation management provider is a m
 
 **Automate** enables real-time, bidirectional process integration. This embedded automate feature provides you with a method to implement adaptive automation for management events and administrative or operational activities.
 
+**Embedded Workflows** provide the capability to simplify service catalog creation and improve the overall service catalog experience through a step-based automation system. Each step within a workflow runs a container to complete processes. Workflows are imported from GitHub repository content into {{ site.data.product.title_short }} where they are made available for users to run. *Embedded Workflows* are an alternative to the *Embedded Automate* capability for authoring and running automation. With *Embedded Workflows*, you can continue to use the *Embedded Automate* feature for service catalog creation and management.
```

Comment:
> Workflows are imported from GitHub repository content

We shouldn't specify GitHub specifically, any git repository will work

Reply:
This is updated to be just Git repository.